test/parallel/test-regress-GH-1531.js

```diff
@@ -1,16 +1,19 @@
 'use strict';
 const common = require('../common');
 
+// This test ensures that a http request callback is called
+// when the agent option is set. See https://github.com/nodejs/node-v0.x-archive/issues/1531
```

Comment:
Would you mind moving the `See ...` to a new line to stay under 80 characters.

Reply:
@cjihrig No problem at all!

Reply:
Thanks!